src/firefly/java/edu/caltech/ipac/firefly/visualize/ClientFitsHeader.java

```diff
@@ -81,7 +81,7 @@ public Iterator<String> iterator() {
     public double getCDelt2() { return getDoubleHeader(CDELT2); }
     public double getBScale() { return getDoubleHeader(BSCALE); }
     public double getBZero() { return getDoubleHeader(BZERO); }
-    public double getBlankValue() { return getDoubleHeader(BLANK_VALUE); }
+    public double getBlankValue() { return getDoubleHeader(BLANK_VALUE, Double.NaN); }
```

Comment:
I see why this works - essentially we're eliminating the default for `BLANK` - but for the record there's another problem here.   `BSCALE` and `BZERO` are genuinely floating-point parameters (this is explicit in the standard; section 4.4.2.5) but the value of `BLANK` is specified to be an integer:

[FITS 4.0.0](https://fits.gsfc.nasa.gov/standard40/fits_standard40aa-le.pdf) section 4.4.2.5:

> **BLANK keyword.** (...) The value field shall contain an integer that specifies the value that is used within the integer array to represent pixels that have an undefined physical value.

Treating `BLANK` as a double inside Firefly has the advantage that you don't have to maintain a separate flag for "`BLANK` was not specified", but it means that if `BLANK` were ever set to a value that can't be represented exactly in a double, this code would fail.  I *think* this could only happen for 64-bit integer images (`BITPIX=64`), so perhaps this is a safe risk to take -- but formally it is an error.